### ISSUE TITLE
Error: Can't end BackgroundTask: no background task exists with identifier

### ISSUE THREAD

**Issue by proxi** [firebase/firebase-ios-sdk#3625]
### Environment

  * Xcode version: 10.3
  * Firebase SDK version: 6.6.0

### Problem

A breakpoint fires and a console warning is shown:

```
Can't end BackgroundTask: no background task exists with identifier 344 (0x158), or it may have already been ended. Break in UIApplicationEndBackgroundTaskError() to debug.
```

<img width="1020" alt="Zrzut ekranu 2019-08-19 o 10 07 31" src="https://user-images.githubusercontent.com/107399/63249688-431ff300-c26a-11e9-907b-625ba93d5671.png">

#### Steps to reproduce:

I couldn't figure out the exact reproducible steps. It seems to happen quite a lot during development.

I suspect it is related to the way how `_backgroundID` is stored in `GDTCCTUploader`.

**Comment by mikehaney24** [comment]
Hopefully #3627 addresses this, but it likely won't go out in the next update.

**Comment by paulb777** [comment]
#3627 will make 6.7.0

**Comment by mikehaney24** [comment]
Still an issue after my PR, working on it.

**Comment by mikehaney24** [comment]
This should be addressed in the newly release Firebase 6.7.0, except in iOS 13. There may be an OS-level bug causing the warning.
